contrib/managed-azure/setup_aks_cluster.sh

```diff
@@ -20,8 +20,6 @@ az group create \
 --name ${AKS_RG} \
 --location ${LOCATION}
 
-az identity create --name $AKS_CP_MI_NAME --resource-group $PERSISTENT_RG_NAME
```

Comment:
This shouldn't cause issues but is just unnecessary since the MI is made in a persistant RG